config/locales/en.yml

```diff
@@ -19,7 +19,6 @@ en:
       wrong_length: wrong length (should be %{count} characters).
       too_young: You must be 18 on or before November 3, 2020 to vote in this election.
   or: "OR"
-  required: "Required"

```

Comment:
Is there a reason this was removed?

Reply:
We use it explicitly e.g. https://github.com/TrustTheVote-Project/Rocky-OVR/blob/production/app/helpers/application_helper.rb#L144

Reply:
it's a duplicate of line 12